### DISCUSSION TITLE
image is not visible as describe in documentation.

### DISCUSSION THREAD

**Discussion by ArsalanSiddique** [spatie/laravel-medialibrary#2760] [Q&A]
Hello developers

As you define in docs. 
`$yourModel->getMedia('images')->first()->getUrl('thumb');`
it returns

```
Spatie\MediaLibrary\MediaCollections\Exceptions\InvalidConversion
There is no conversion named `thumb` (View: /var/www/html/blog/resources/views/products/show.blade.php)
```
not working but when I try this, it works fine.

`$myModel->getFirstMediaUrl('images', 'thumb')`
`$product->getMedia('images')->first()->getUrl() `

Today I use your media library for the first time, Would you explain to me why your defined official docs. code not working?

**Comment by squiaios** [comment]
https://github.com/spatie/laravel-medialibrary/blob/b7929b030ec4b10ec023bae553f69f0252f63ee7/src/InteractsWithMedia.php#L300

A check is made to see if the conversion has been made. In your case it's not the case :)
